.gitignore

```diff
@@ -15,4 +15,5 @@ doxygen/
 workspace.xml
 java_console_binary
 .vscode
-*.json
+*.cmp
```

Comment:
i believe this change to .gitignore should not be part of "E30 M3 Tach output solution" pull request

we can discuss it as a separate change if it's needed